.gitignore

```diff
@@ -0,0 +1,3 @@
+/packages
+**.swp
```

Comment:
Presumably for vim backup but should this be `*.swp` or `**/.swp` or similar?

Reply:
The idea was to get `*.swp` files in any directory (actually from nano, but presumably similar from vim). I think that `**/.swp` would only work if the file was called `.swp` in any dir and `*.swp` would only capture the root dir. I think `**.swp` should be fine capturing the equivalent of `**/*.swp`, but happy to change if desired

Reply:
`*.swp` should handle any directory - I did just check to confirm. It's only a niggle really but in this case the double wildcard is effectively the same as a single. It seems to agree with the example in the docs: https://git-scm.com/docs/gitignore#_examples